### DISCUSSION TITLE
Numbers of the Energy Dashboard are a lot different than those, reported by RCT Portal

### DISCUSSION THREAD

**Discussion by peterge1998** [weltenwort/home-assistant-rct-power-integration#326] [General]
![image](https://github.com/weltenwort/home-assistant-rct-power-integration/assets/47355238/41f58c8f-c76b-47a1-98f6-eb694a34a900)

I just set up the integration successfully and followed all steps to set up the Energy Dashboarrd. But the numbers in Home Assistant are quite different, from those, reported by RCT Portal.
Did I set it up correctly, or is this just a conversion thing?

![image](https://github.com/weltenwort/home-assistant-rct-power-integration/assets/47355238/fe90b75e-51cc-4114-8d6c-952d6df0dd5e)

**Comment by mosandlt** [comment]
same for me. it shows the wrong values as there are the total numbers. i can not add the right values. Please support us

**Comment by mosandlt** [comment]
numbers are correct btw. these are just sums for the full day

I would close this topic
![Bildschirmfoto 2024-12-18 um 10 20 46](https://github.com/user-attachments/assets/18db3aa4-01c0-40e9-924b-93f59a22d580)
![Bildschirmfoto 2024-12-18 um 10 20 53](https://github.com/user-attachments/assets/80fe2155-c941-4afd-9b5b-ae8101a0da17)
